frontend/src/App.js

```diff
@@ -234,11 +235,11 @@ AND refs.ref_name = 'HEAD'`
     );
 
     api
-      .query(sql)
+      .query(sql, abortController.signal)
       .then(response => {
-        this.setResult(key, { sql, response });
+        this.setResult(key, { sql, response, abortController });
```

Comment:
why do you need abort controller after request is finished already?

Reply:
I would really prefer to remove it. You have a check for the existence of abortController in cancel function. I spent quite a while to find for which case it might be needed but apparently it's not. I think removing redundant code makes it easier to read & understand.

Reply:
done in 359d4de